src/libraries/Microsoft.PowerFx.Interpreter/Environment/PowerFxConfigExtensions.cs

```diff
@@ -3,7 +3,10 @@
 
 using System;
 using System.Collections.Generic;
+using System.Text;
+using Microsoft.PowerFx.Core;
```

Comment:
Those new using statements are probably not needed

Reply:
Absolutely true :)